Orange/util.py

```diff
@@ -246,6 +248,13 @@ def __repr__(self):
                                  ', '.join('{}={!r}'.format(*pair)
                                            for pair in names_values))
 
+    @staticmethod
+    def __equal(obj1, obj2):
+        try:
+            return obj1.size == obj2.size and np.all(obj1 == obj2)
+        except AttributeError:
+            return np.all(obj1 == obj2)
```

Comment:
Adding an explicit test (or more) for __equal might help :)

Although, you might have to consider renaming it to _equal as testing __private stuff is tricky.